### PR TITLE
.coafile: Check all python files

### PR DESCRIPTION
**Pull request by jayvdb**

### CODE REVIEW

File: .ci/deploy.sh [hunk 1]
```diff
@@ -4,19 +4,14 @@ set -e # Exit with nonzero exit code if anything fails
...
 # Save some useful information
 REPO=`git config remote.origin.url`
 SSH_REPO=${REPO/https:\/\/github.com\//git@github.com:}
 SHA=`git rev-parse --verify HEAD`
 
-# Clone the existing gh-pages for this repo into out/
-# Create a new empty branch if gh-pages doesn't exist yet (should only happen on first deply)
+# Clone the existing gh-pages for this repo into out/ .
+# Create a new empty branch if gh-pages doesn't exist yet
+# (should only happen on first deply)
```
Comment (adtac) [inline .ci/deploy.sh]: s/deply/deploy

might as well fix it lol

File: .coafile [hunk 2]
```diff
@@ -1,27 +1,33 @@
 [all]
-files = *.py, community/**/*.py, gci/**/*.py, activity/*.py
-ignore = gci/client.py, gci/migrations/**, private/**, gsoc/migrations/**
+files = **.py, **.js, **.js, **.sh
```
Comment (adtac) [inline .coafile]: double-check all `.js` files, eh? :)

**Comment by adtac** [comment]
ack https://github.com/coala/community/pull/92/commits/768b6a9cb43f713aa5ece682d60db39f8e930449 https://github.com/coala/community/pull/92/commits/708c8b0bc7430308677a9659aabb1c014f81e9e0

**Comment by adtac** [comment]
ack https://github.com/coala/community/commit/768b6a9cb43f713aa5ece682d60db39f8e930449 https://github.com/coala/community/commit/708c8b0bc7430308677a9659aabb1c014f81e9e0

**Comment by jayvdb** [comment]
reack 768b6a9 708c8b0

**Comment by jayvdb** [comment]
ack 768b6a9 708c8b0

**Comment by adtac** [comment]
ack https://github.com/coala/community/commit/768b6a9cb43f713aa5ece682d60db39f8e930449 https://github.com/coala/community/commit/708c8b0bc7430308677a9659aabb1c014f81e9e0

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
reack https://github.com/coala/community/pull/92/commits/fd5cb09df9737416697ba4660f3fe110f1e8fd43 https://github.com/coala/community/pull/92/commits/e565fd90364580fc26a2d825569944581ddc06e8

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**